src/scorecard.js

```diff
@@ -0,0 +1,73 @@
+var Scorecard = function(content) {
+  this.frameRecord = [];
+  this.totals = [];
+  this.DEFAULT_FRAMES = 10;
+  this.currentTurn = 0;
+  this.maxTurns = 20;
+  for (var i = 0; i < this.DEFAULT_FRAMES; i++) {
+    this.frameRecord.push(new content);
+  };
+};
+
+Scorecard.prototype.updateTotal = function() {
+  if (this.frameRecord[this.getCurrentFrame()].strike === true || this.frameRecord[this.getCurrentFrame()].spare === true) {
+    this.totals.push(10);
```

Comment:
This whole conditional is unnecessary.  Scorecard is responsible for keeping the running score from the frames - the Frame is responsible for the interpreting the result of each frame.  Simply setting `result` to 10 in your frame for a spare or score would remove the need for this if statement and correctly encapsulate this feature.